java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -933,1212 +345,26 @@ public static ColumnVector sequence(Scalar initialValue, int rows) {
-   *               %f 	6-digit microsecond: 000000-999999
-   *               See https://github.com/rapidsai/custrings/blob/branch-0.10/docs/source/datetime.md
-   *
-   * Reported bugs
-   * https://github.com/rapidsai/cudf/issues/4160 after the bug is fixed this method should
-   * also support
-   *               %I 	12-hour of the day: 01-12
-   *               %p 	Only 'AM', 'PM'
-   *               %j   day of the year
-   *
-   * @return A new vector allocated on the GPU
-   */
-  public ColumnVector asStrings(String format) {
-    assert type.isTimestamp() : "unsupported conversion from non-timestamp DType";
-    assert format != null || format.isEmpty(): "Format string may not be NULL or empty";
-
-    return new ColumnVector(timestampToStringTimestamp(this.getNativeView(), format));
-  }
-
-  /////////////////////////////////////////////////////////////////////////////
-  // LISTS
-  /////////////////////////////////////////////////////////////////////////////
-
-  /**
-   * For each list in this column pull out the entry at the given index. If the entry would
-   * go off the end of the list a NULL is returned instead.
-   * @param index 0 based offset into the list. Negative values go backwards from the end of the
-   *              list.
-   * @return a new column of the values at those indexes.
-   */
-  public ColumnVector extractListElement(int index) {
-    assert type == DType.LIST : "A column of type LIST is required for .extractListElement()";
-    return new ColumnVector(extractListElement(getNativeView(), index));
-  }
-
-  /////////////////////////////////////////////////////////////////////////////
-  // STRINGS
-  /////////////////////////////////////////////////////////////////////////////
-
-  /**
-   * Convert a string to upper case.
-   */
-  public ColumnVector upper() {
-    assert type == DType.STRING : "A column of type string is required for .upper() operation";
-    return new ColumnVector(upperStrings(getNativeView()));
-  }
-
-  /**
-   * Convert a string to lower case.
-   */
-  public ColumnVector lower() {
-    assert type == DType.STRING : "A column of type string is required for .lower() operation";
-    return new ColumnVector(lowerStrings(getNativeView()));
-  }
-
-  /**
-   * Concatenate columns of strings together, combining a corresponding row from each column
-   * into a single string row of a new column with no separator string inserted between each
-   * combined string and maintaining null values in combined rows.
-   * @param columns array of columns containing strings.
-   * @return A new java column vector containing the concatenated strings.
-   */
-  public ColumnVector stringConcatenate(ColumnVector[] columns) {
-    try (Scalar emptyString = Scalar.fromString("");
-         Scalar nullString = Scalar.fromString(null)) {
-      return stringConcatenate(emptyString, nullString, columns);
-    }
+  public final ColumnVector stringConcatenate(ColumnVector[] columns) {
```

Comment:
Shouldn't these input be ColumnView instead of ColumnVector? And why is it not static?  I know it was not static before but that really looks like a bug that we can easily fix.